packages/reason-expo/src/SQLite.re

```diff
@@ -82,5 +82,5 @@ module Database = {
     "transaction";
 };
 
-[@bs.module "expo-sqlite"] [@bs.scope "SQLite"]
+[@bs.module "expo-sqlite"]
```

Comment:
are you supposed to remove bs.scope? if so this looks good

Reply:
removed [@bs.scope "SQLite"] because failed on testing. Works like this.